### PR TITLE
V1 readme

### PR DESCRIPTION
**Pull request by saumilptl**

### CODE REVIEW

**Comment by saumilptl** [comment]
@davidscodex @derrickpersson any additional information we should provide here?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,54 @@
...
+
+1. Visit <https://app.echolayer.com> to create your organization with EchoLayer. We only ask for a name for your org. Yes, that’s really it.
+
+    `👉 You’ll need to authenticate with GitHub prior to this. We only ask for your email address so we can create a user account in EchoLayer for you.`
+
+2. Once created, you’ll be redirected to the homepage for your organization in EchoLayer. Click on the link to visit the [API Key page](https://app.echolayer.com/app/account/api-keys)
+    1. Generate a new API key
+    2. Click the copy button
+    3. Paste the value of the token temporarily somewhere you’ll have access to as you’ll need it momentarily (or just leave this open so you can come back and copy it)
+
```
Comment (davidscodex) [inline README.md:47]: 3. Add the apiKey to a Repository Secret named ECHOLAYER_API_KEY

**Review by davidscodex — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,10 +1,52 @@
...
+jobs:
+  echo_layer_pipeline:
+    runs-on: ubuntu-latest
+    name: Import Backstage Catalog to EchoLayer
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Sync Backstage Catalog
+        uses: usecodex/echolayer-pipeline-action@HEAD
```
Comment (saumilptl) [inline README.md:18]: @davidscodex we'll go with HEAD for now but in the future we'll likely want users to point to specific release versions.

Comment (davidscodex) [reply README.md:18]: Yep

**Pull request merged**